js/views/questionView.js

```diff
@@ -310,6 +309,11 @@ class QuestionView extends ComponentView {
     this.$('.component__widget').removeClass('is-submitted');
   }
 
+  focusAfterReset() {
+    // Focus on the first readable item in this element
+    a11y.focusNext(this.$el, { preventScroll: true });
```

Comment:
I'm not sure you want this: `{ preventScroll: true }`

Reply:
Removed. I agree that it would make sense to scroll to the point where the new focus is. Otherwise, it could be off screen and the user could lose their place.

Do you know why this was there in the first place? Was it to avoid taking control away from users who use assistive technologies? Is there an a11y concern about controlling focus when the user prefers reduced motion perhaps? Can't seem to find any a11y guidelines about this.

Reply:
All of the preventScroll behaviours are just to prevent jumping for sighted users. But in this case, I don't think it makes sense. We'll see.

> Was it to avoid taking control away from users who use assistive technologies? 

No, just to prevent jumping.
 
> Is there an a11y concern about controlling focus when the user prefers reduced motion (ex. smooth scrolling could still pose an issue for these users)? 

No, reduced motion is predominantly about aesthetic animations not functional jumps, I believe.